src/features/boards/PostCard.tsx

```diff
@@ -0,0 +1,72 @@
+import Icon from "@/components/icon";
+import Formatter from "@/utils/formatter";
+import clsx from "clsx";
+import Image from "next/image";
+
+interface Writer {
+  id: number;
+  nickname: string;
+}
+
+interface PostCardProps {
+  isPopular?: boolean;
+  image?: string;
+  title: string;
+  content?: string;
+  likeCount: number;
+  createdAt: string;
+  writer: Writer;
+}
+
+export default function PostCard({
+  isPopular,
+  image,
+  title,
+  writer,
+  createdAt,
+  content,
+  likeCount,
+}: PostCardProps) {
+  const formattedDate = Formatter(createdAt);
+  return (
+    <div
+      className={clsx(
+        "flex h-40 w-[340px] flex-col gap-1.5 rounded-[20px] border border-border-primary bg-background-primary p-3 tablet:h-44 tablet:w-[620px] tablet:p-4 desktop:w-[552px]",
+        isPopular ? "justify-between" : "justify-around"
+      )}
+    >
+      {isPopular && (
+        <div className="flex h-7 w-16 items-center justify-center rounded-full bg-background-secondary px-1 py-1.5 text-brand-primary">
+          <Icon name="best" />
+          인기
+        </div>
+      )}
+      <div className="flex flex-col gap-2">
+        <div className="flex h-20 w-[308px] justify-between tablet:h-[88px] tablet:w-[572px] desktop:w-[504px]">
+          <div className="flex flex-col gap-2">
+            <h3 className="text-lg-b tablet:text-2lg-b">{title}</h3>
+            <p className="text-sm-m tablet:text-md-r">{content}</p>
```

Comment:
지금은 content 내용을 전부 보여주고 있는 것 같은데요,
content 내용이 많을 경우를 고려해 `line-clamp`를 적용해도 좋을 것 같습니다.
```suggestion
            <p className="text-sm-m tablet:text-md-r line-clamp-2">{content}</p>
```

Reply:
오.. 아예 생각 못 한 부분인데 감사합니다 👍